sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -912,6 +912,12 @@ qualifiedNameList
     : qualifiedName (',' qualifiedName)*
     ;
 
+functionCallName
```

Comment:
nit: `functionName` should be good enough, in case we want to use it in `CREATE FUNCTION` in the future.

Reply:
Thanks, done in d357a9a.